apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_details/TradeDetailsController.java

```diff
@@ -0,0 +1,150 @@
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.desktop.main.content.bisq_easy.open_trades.trade_details;
+
+import java.util.Optional;
+
+import bisq.bisq_easy.NavigationTarget;
+import bisq.chat.bisqeasy.open_trades.BisqEasyOpenTradeChannel;
+import bisq.common.monetary.Coin;
+import bisq.common.monetary.Fiat;
+import bisq.common.monetary.Monetary;
+import bisq.common.network.AddressByTransportTypeMap;
+import bisq.common.util.StringUtils;
+import bisq.contract.bisq_easy.BisqEasyContract;
+import bisq.desktop.common.view.Controller;
+import bisq.desktop.common.view.InitWithDataController;
+import bisq.desktop.common.view.NavigationController;
+import bisq.desktop.overlay.OverlayController;
+import bisq.desktop.ServiceProvider;
+import bisq.network.identity.NetworkId;
+import bisq.presentation.formatters.AmountFormatter;
+import bisq.presentation.formatters.DateFormatter;
+import bisq.presentation.formatters.PriceFormatter;
+import bisq.trade.bisq_easy.BisqEasyTrade;
+import bisq.trade.bisq_easy.BisqEasyTradeFormatter;
+import bisq.trade.bisq_easy.BisqEasyTradeUtils;
+import bisq.user.identity.UserIdentityService;
+import com.google.common.base.Joiner;
+import java.util.stream.Collectors;
+import javafx.beans.property.SimpleStringProperty;
+import lombok.EqualsAndHashCode;
+import lombok.extern.slf4j.Slf4j;
+import lombok.Getter;
+import lombok.ToString;
+
+@Slf4j
+public class TradeDetailsController extends NavigationController
+        implements InitWithDataController<TradeDetailsController.InitData> {
+    @Getter
+    @EqualsAndHashCode
+    @ToString
+    public static class InitData {
+        private final BisqEasyTrade bisqEasyTrade;
+        private final BisqEasyOpenTradeChannel channel;
+
+        public InitData(BisqEasyTrade BisqEasyTrade, BisqEasyOpenTradeChannel channel) {
+            this.bisqEasyTrade = BisqEasyTrade;
+            this.channel = channel;
+        }
+    }
+
+    @Getter
+    private final TradeDetailsView view;
+
+    @Getter
+    private final TradeDetailsModel model;
+    protected final UserIdentityService userIdentityService;
+
+    public TradeDetailsController(ServiceProvider serviceProvider) {
+        super(NavigationTarget.BISQ_EASY_TRADE_DETAILS);
+
+        userIdentityService = serviceProvider.getUserService().getUserIdentityService();
+        model = new TradeDetailsModel();
+        view = new TradeDetailsView(model, this);
+    }
+
+    @Override
+    public void initWithData(InitData initData) {
+        BisqEasyTrade trade = initData.getBisqEasyTrade();
+        BisqEasyOpenTradeChannel channel = initData.channel;
+        model.setTradeId(new SimpleStringProperty(trade.getId()));
```

Comment:
We prefer to use final fields in the model for properties. Not a big problem to do it like that but better to stick with the established style.